### DISCUSSION TITLE
FritzBox7690 funktioniert, aber nur über einen unschönen Umweg

### DISCUSSION THREAD

**Discussion by catrinelc** [Freetz-NG/freetz-ng#1011] [Allgemein]
Hallo zusammen

Ich habe leider nicht gewusst dass, die Issues vermutlich nicht bearbeitet werden,
deswegen jetzt bringe ich die Issue 1010 in den Feedbacks:

https://github.com/Freetz-NG/freetz-ng/issues/1010

Danke
CC

**Comment by fda77** [comment]
Ich hab das Issue in eine Diskussion umgewandelt: #1012
Neue Issues kann man leider nicht deaktivieren, daher werden die automatisch geschlossen.
Zum einen fand ich die Issue/Diskussion wechsel immer blöd
Ausserdem haben bei Issues viele die Erwartung dass es jemden gibt der auf jeden Fall das issue behebt, und es ging recht wenige dies dies tun
